README.md

```diff
@@ -118,28 +135,90 @@ sudo adduser ccdc --uid=1397
 
 # You will also need to ensure the user "ccdc" has read access to any in-house or CSP databases.
-# E.g. if these are in the userdata directory:
-sudo chown -R ccdc:ccdc userdata/ 
+# E.g. if these are in the csd-data directory:
+sudo chown -R ccdc:ccdc csd-data/ 
 
-# Use one of the following commands
+# For testing use one of the following commands.
+# These do not load in-house databases and the user access control is disabled.
+# These features will be configured later.
 
-#Use this command if you have no in-house databases and don't want to use macromolecule hub
-docker compose -f docker-compose.yml -f docker-compose.ssl.yml up -d
+# If you are not using Macromolecule Hub
+docker compose -f docker-compose.yml -f docker-compose.ssl.yml -f docker-compose.disable-user-access.yml up -d
 
-#Use this command if you have in-house databases and don't want to use macromolecule hub 
-docker compose -f docker-compose.yml -f docker-compose.db-config.yml -f docker-compose.ssl.yml up -d
+# If you are using Macromolecule Hub
+docker compose -f docker-compose.yml -f docker-compose.ssl.yml -f docker-compose.disable-user-access.yml -f docker-compose.macromolecule-hub.yml up -d
+```
 
-#Use this command if you have in-house databases and want macromolecule hub 
-docker compose -f docker-compose.yml -f docker-compose.macromolecule-hub.yml -f docker-compose.db-config.yml -f docker-compose.ssl.yml up -d
+CHECK: To check that the install has completed and that all the services are running, run:
 
-#Use this command if you have no in-house databases and want macromolecule hub
-docker compose -f docker-compose.yml -f docker-compose.macromolecule-hub.yml -f docker-compose.ssl.yml up -d
+```sh
+docker compose ps
 ```
 
+which should show the state of the services to all be `Up`. If any services have the state `Up (unhealthy)` or `Exit` then restart the stack (`docker compose down` then repeat
+the `docker compose ... up -d` command run earlier). If the issues persist then please contact CCDC Support.
+
+For more information see the [Docker volumes documentation](https://docs.docker.com/compose/compose-file/#volumes).
+
+To access the On-Site Lattice and WebCSD service locally go to <https://full.server.hostname> in a browser,
+replacing `full.server.hostname` by the full hostname of your Docker host. Check that you can search the CSD.
+If you have problems check that `PUBLIC_URI` has been set to <https://full.server.hostname> in `.env`.
+
+## Further Configuration
+
+Your On-Site WebCSD server is now up and running but you will probably need to configure user access control
+and in-house databases. This is done via docker compose files
+(see [WebCSD Configuration Files](https://github.com/ccdc-opensource/on-site-webcsd/wiki/WebCSD-configuration-files) in the wiki).
+After making any changes to these files, restart the stack using the following config files.
+
+- docker-compose.yml
+- docker-compose.ssl.yml
+- Any other docker compose files which you have customised
```

Comment:
This could also include the MMHUB file. 

I think this section repeats the one above that explains which files should be included in your compose file. Could we reference that section instead, instructing to then append any customised compose files?

Reply:
I've updated it to say "restart the stack as described above, adding in any docker compose files which you have customised".